provider/internal/test/suite.go

```diff
@@ -69,33 +70,41 @@ func (s *IntegrationTestSuite) SetupSuite() {
 		operatorPort, _ := testutils.GetFreePort()
 
 		s.ProviderConfig = s.ProviderConfigFn(dataStore)
-		valkyrieConfig := configs.ValkyrieConfig{
-			Logging: configs.LogConfig{
-				Level: "fatal",
-			},
-			ProviderBasePath: "/providers",
-			Providers: []configs.ProviderConf{
-				s.ProviderConfig,
-			},
-			Pam: configs.PamConf{
-				"name":    "generic",
-				"url":     pamURL,
-				"api_key": dataStore.GetPamAPIToken(),
-			},
-			HTTPServer: configs.HTTPServerConfig{
-				ProviderAddress: fmt.Sprintf(baseAddr, providerPort),
-				OperatorAddress: fmt.Sprintf(baseAddr, operatorPort),
-			},
-		}
-
-		s.valkyrie = server.NewValkyrie(context.TODO(), &valkyrieConfig)
+		valkyrieConfig := s.buildConfig(pamURL, providerPort, operatorPort, dataStore)
 
+		if v, err := server.NewValkyrie(context.TODO(), &valkyrieConfig); err != nil {
+			log.Error().Err(err)
```

Comment:
I might be wrong here, but I think it needs a message (or call to `Send()`) to actually log the error

Reply:
You're probably right. But also realised the error is already logged inside `server.NewValkyrie` so i'll put something else here